contracts/interfaces/IRewardEthToken.sol

```diff
@@ -26,8 +26,8 @@ interface IRewardEthToken is IERC20Upgradeable {
     * @param reward - user reward checkpoint.
     */
     struct Checkpoint {
-        uint256 rewardPerToken;
-        uint256 reward;
+        uint128 reward;
+        uint64 rewardPerToken;
```

Comment:
Why 64-bit instead of 128-bit is used here?  Also, 64-bit can represent only up to ~18 ether.  `rewardPerToken` can grow more than 18 ether, no?

Reply:
Not sure if it can grow more than 18 ether, as it would be too high reward per every stETH.
The reason why it's 64-bit instead of 128-bit is because the storage update for `updateTotalRewards` function takes requires only 2 slots instead of 3.